### ISSUE TITLE
wrap_root on PUT or POST doesn't do anything

### ISSUE THREAD

**Issue by noctivityinc** [flexirest/flexirest#105]
After reading issue #87 we tried the following:
```
  put :save, "/admin/sites/:id", wrap_root: 'site'
  post :create, "/admin/sites", wrap_root: 'site'
```
in our Site model but when we check the body, nothing is wrapped:

```site.save
  Flexirest Site#save - Requesting http://api.pa.local:3001/v1/admin/sites/2
Flexirest Verbose Log:
  Request
  >> PUT /v1/admin/sites/2 HTTP/1.1
  >> Accept : application/hal+json, application/json;q=0.5
  >> Authorization : Bearer: eyJhbGciOiJIUzI1NiJ9.eyJhZG1pbl9pZCI6MX0.z4mFEwWrc-vXIq48vHdk-eA9b5b29_y9Kh4fyYN24LI
  >> Content-Type : application/x-www-form-urlencoded
  >> Body:
code=testtest&created_at=2018-10-04T16%3A25%3A13%2B00%3A00&description=&name=Testing&status=pending&updated_at=2018-10-04T16%3A25%3A13%2B00%3A00```

**Comment by noctivityinc** [comment]
Ok - we were able to solve this by changing the `request_body_type :json` from form, but that requirement is not in the docs.  Is there a reason that you didn't add the `wrap_root` functionality for forms as well?

Thanks!

**Comment by andyjeffries** [comment]
Wasn't intentionally excluded, just wasn't added to there. So you'd expect it to be:

```
site[code]=testtest&site[created_at]=2018...
```

Right?

**Comment by andyjeffries** [comment]
This has been implemented now and has been released as v1.7.3. Thanks for the issue.
